### PR TITLE
Note that Socket module requires netstat

### PR DESCRIPTION
**Pull request by ashb**
It's available in most places, but it might not be available in docker containers so add a clarifying note to the docs -- save someone a bit of time if they get this error:

```
ipdb> Socket("tcp://80").is_listening
*** Failed: Unexpected exit code 127 for CommandResult(command=b'netstat -n -l -t', exit_status=127, stdout=b'', stderr=b'/bin/sh: 1: netstat: not found\n')
```

### CODE REVIEW

**Comment by philpep** [comment]
Ok to add a note. But the error message is clear enough no ? "netstat: not found"

**Comment by philpep** [comment]
Ok didn't see this was a PR. Thanks ! I restarted the build that has random failed due to debian mirrors issues.

File: testinfra/modules/socket.py [hunk 1]
```diff
@@ -68,6 +68,9 @@ class Socket(Module):
     """Test listening tcp/udp and unix sockets
 
     ``socketspec`` must be specified as ``<protocol>://<host>:<port>``
+    
+    This module needs the  ``netstat`` binary to be installed in the
+    target environment otherwise it will not funciton.
```
Comment (philpep) [inline testinfra/modules/socket.py]: s/funciton/function

But I'd reword like this "This module require the ``netstat`` command on target host".

**Comment by ashb** [comment]
@philpep reworded and force pushed.
